apps/api/src/models/rules/fees/fee-ratio-initial-investment.ts

```diff
@@ -27,18 +30,30 @@ export class FeeRatioInitialInvestment extends Rule<Settings> {
       : 0;
 
     if (feeRatio > ruleSettings.thresholdMax) {
+      const evaluation = this.i18nService
+        .getTranslation({
+          id: 'rule.fee-ratio-initial-investment.exceed',
+          languageCode: DEFAULT_LANGUAGE_CODE
```

Comment:
You could move the replace logic into `getTranslation()` to make it easier for the consumer:
```
this.i18nService.getTranslation({
  id: 'rule.fee-ratio-initial-investment.exceed',
  languageCode: DEFAULT_LANGUAGE_CODE,
  placeholders: {
    feeRatio: (feeRatio * 100).toPrecision(3),
    thresholdMax: (ruleSettings.thresholdMax * 100).toFixed(2)
  }
});
```